test/functional/index.js

```diff
@@ -29,13 +29,13 @@ define(function (require) {
     });
 
     const apps = [
-      'intern/dojo/node!./apps/xpack',
-      'intern/dojo/node!./apps/discover',
-      'intern/dojo/node!./apps/management',
-      'intern/dojo/node!./apps/visualize',
-      'intern/dojo/node!./apps/console',
+      // 'intern/dojo/node!./apps/xpack',
+      // 'intern/dojo/node!./apps/discover',
+      // 'intern/dojo/node!./apps/management',
+      // 'intern/dojo/node!./apps/visualize',
+      // 'intern/dojo/node!./apps/console',
       'intern/dojo/node!./apps/dashboard',
-      'intern/dojo/node!./status_page'
+    //  'intern/dojo/node!./status_page'
```

Comment:
This time it was actually intentional :)  Trying to reduce the time for a build because I was having a very tough time figuring out what the issue was (turns out it was a delay between saving a new dashboard and reloading the url to point to the newly saved dashboard).